### PR TITLE
Fix text coverage

### PR DESCRIPTION
**Pull request by bastien-phi**
Fixes https://github.com/pestphp/pest/issues/499

Bump paratest to ^6.4.2 as the color option in CoverageReporter::text was added in this version

### CODE REVIEW

File: composer.json
Comment (owenvoke) [inline composer.json:21]: @nunomaduro, should this be changed? 🤔 Not entirely sure.

**Review by owenvoke — APPROVED** [review]
LGTM! Just one query, but otherwise is good.

File: composer.json
Comment (bastien-phi) [reply composer.json:21]: I needed to change that because composer complained at install when using `pest/pest:dev-master`
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - brianium/paratest v6.4.3 requires phpunit/phpunit ^9.5.14 -> satisfiable by phpunit/phpunit[9.5.14, ..., 9.5.x-dev].
    - brianium/paratest v6.4.2 requires phpunit/phpunit ^9.5.13 -> satisfiable by phpunit/phpunit[9.5.13, ..., 9.5.x-dev].
    - You can only install one version of a package, so only one of these can be installed: phpunit/phpunit[dev-master, 9.5.13, ..., 9.5.x-dev].
    - phpunit/phpunit 10.0.x-dev is an alias of phpunit/phpunit dev-master and thus requires it to be installed too.
    - pestphp/pest dev-master requires phpunit/phpunit 10.0.x-dev -> satisfiable by phpunit/phpunit[10.0.x-dev (alias of dev-master)].
    - Root composer.json requires pestphp/pest dev-master -> satisfiable by pestphp/pest[dev-master].
    - Root composer.json requires brianium/paratest ^6.4.2 -> satisfiable by brianium/paratest[v6.4.2, v6.4.3].
```

Comment (owenvoke) [reply composer.json:21]: Ah, that makes sense. I guess everything is geared up for Pest 2.x 🤔

Comment (owenvoke) [reply composer.json:21]: I did think that, does it have a branch alias for 1.x?

Comment (fabio-ivona) [reply composer.json:21]: hmm, seems not

image

but fixes can be tagged on that branch

Comment (owenvoke) [reply composer.json:21]: @fabio-ivona, I've updated the branch alias on the `1.x` branch. @bastien-phi, would you be able to update this to `dev-1.x`? 🤔

Comment (bastien-phi) [reply composer.json]: can't use dev-1.x but used 1.x-dev

```
  Problem 1
    - Root composer.json requires pestphp/pest dev-1.x, found pestphp/pest[dev-higher-order-test-expect-escape, dev-filtered_datasets, dev-master, dev-next, dev-repeat, dev-revert-462-revert-461-json_fix, dev-better-tap-type-hinting, dev-performs_no_expectations, dev-parallel-highligher-support, dev-parallel, v0.1.0, ..., v0.3.19, v1.0.0, ..., 1.x-dev, 2.x-dev (alias of dev-next)] but it does not match the constraint.
```

**Comment by bastien-phi** [comment]
Static analysis still fails but looks like it is not related to this PR

**Review by owenvoke — APPROVED** [review]

File: composer.json
Comment (owenvoke) [reply composer.json]: Yeah, my bad. Got the alias the wrong way round. 👍🏻

Comment (bastien-phi) [reply composer.json]: Yeah indeed... 🤦
I will change that. Good catch !

Comment (bastien-phi) [reply composer.json]: @fabio-ivona Done !

Comment (owenvoke) [reply composer.json]: We'll need to merge this back into `master` after as well then. 👍🏻 As both 2.x and 1.x will need this fix.

**Comment by owenvoke** [comment]
I'll fix the static analysis issues once this is merged. @fabio-ivona, all yours. 👍🏻

**Comment by fabio-ivona** [comment]
LGTM, merged

**Pull request merged**
